resources/views/backend/layouts/partials/sidebar-menu.blade.php

```diff
@@ -28,210 +26,20 @@
-                        x-transition:leave-end="opacity-0 max-h-0"
-                        class="submenu pl-12 mt-2 space-y-2 overflow-hidden">
-                        @if ($user->can('settings.edit'))
-                            <li>
-                                <a :style="`color: ${textColor}`" href="{{ route('admin.settings.index') }}"
-                                    class="hover:menu-item-active block px-4 py-2 rounded-lg {{ Route::is('admin.settings.index') ? 'menu-item-active' : 'menu-item-inactive' }}">
-                                    {{ __('General Settings') }}
-                                </a>
-                            </li>
-                        @endif
-                        @canany(['translations.view', 'translations.edit'])
-                            <li>
-                                <a :style="`color: ${textColor}`" href="{{ route('admin.translations.index') }}"
-                                    class="hover:menu-item-active block px-4 py-2 rounded-lg {{ Route::is('admin.translations.*') ? 'menu-item-active' : 'menu-item-inactive' }}">
-                                    {{ __('Translations') }}
-                                </a>
-                            </li>
-                        @endcanany
-                    </ul>
-                </li>
-            @endif
-
-            <!-- Logout Menu Item -->
-            <li class="hover:menu-item-active">
-                <form method="POST" action="{{ route('logout') }}">
-                    @csrf
-                    <button :style="`color: ${textColor}`" type="submit"
-                        class="menu-item group w-full text-left menu-item-inactive">
-                        <img src="{{ asset('images/icons/logout.svg') }}" alt="Roles Icon" class="menu-item-icon dark:invert">
-                        <span class="menu-item-text">{{ __('Logout') }}</span>
-                    </button>
-                </form>
-            </li>
-        </ul>
-    </div>
+    @foreach($menus as $groupName => $groupMenus)
+        {!! ld_apply_filters('sidebar_menu_group_before_' . strtolower($groupName), '') !!}
+        <div>
+            {!! ld_apply_filters('sidebar_menu_group_heading_before_' . strtolower($groupName), '') !!}
+            <h3 class="mb-4 text-xs uppercase leading-[20px] text-gray-400 px-5">
+                {{ __($groupName) }}
+            </h3>
+            {!! ld_apply_filters('sidebar_menu_group_heading_after_' . strtolower($groupName), '') !!}
+            <ul class="flex flex-col mb-6">
+                {!! ld_apply_filters('sidebar_menu_before_all_' . strtolower($groupName), '') !!}
+                {!! $menuService->render($groupMenus, 'textColor') !!}
+                {!! ld_apply_filters('sidebar_menu_after_all_' . strtolower($groupName), '') !!}
+            </ul>
+        </div>
+        {!! ld_apply_filters('sidebar_menu_group_after_' . strtolower($groupName), '') !!}
```

Comment:
need to be like
`sidebar_menu_group_after_main`

`sidebar_menu_group_after_other_menu_group`
means - Other Menu Group -> `_other_menu_group` appended like so, needs strtolower and replace space with underscore.
Handle it from a new method in the service maybe.